Docs/Practitioners/MedicalProfessionalIdentity.md

```diff
@@ -0,0 +1,7 @@
+# MedicalProfessionalIdentity
+
+Holds the federation id of the medical professional
```

Comment:
Provides identity information for a medical professional.

Notice we used medical professional here which would be adverse to the medical practitioner of earlier.